### ISSUE TITLE
Pulmo: Pulmonary langerhans cell histiocytosis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#9754]
**Preferred term label:**
Pulmonary langerhans cell histiocytosis

**Synonyms**
PLCH

**Definition (free text, please give PubMed ID)**
Accumulation of monoclonal CD1a-positive Langerhans cells in the bronchioles and alveolar interstitium that can cause damage to the lungs. 

PMID 20617216: 
Importantly, PLCH pathogenesis is different between childhood and adulthood, because over 95% of affected adults are cigarette smokers [[11](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR11)]. Thus, PLCH is mainly described in adults with a peak frequency at ages 20–40 years. On the other hand, lung involvement in childhood LCH is less often described, due to much lower occurrence [[7](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR7), [12](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR12)–[14](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR14)]. In young adults, computed tomography (CT) scans typically show nodules and cysts, predominantly in the upper and middle lung fields [[11](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR11), [15](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR15)]. Adult PLCH histology is characterized by focal CD1a-positive cells, often organized into granulomas, that infiltrate and destroy distal bronchioles, resulting in cyst formation and pulmonary tissue destruction [[9](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR9)]. The evolution of PLCH in young adults ranges from self-regression, upon discontinuing tobacco consumption, to severe respiratory failure [[16](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR16)]. Secondary pulmonary hypertension can shorten life expectancy. Additionally, lung disease can be life-threatening and might require lung or heart-lung transplantation [[9](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR9), [17](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7487928/#CR17)].

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary interstitial morphology HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Langerhans Cell Histiocytosis (?)  ORPHA:389; MIM: 604856

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
Thanks, added as HP:0034769
